nixos/release.nix

```diff
@@ -244,6 +244,7 @@ in rec {
   tests.containers-macvlans = callTest tests/containers-macvlans.nix {};
   tests.couchdb = callTest tests/couchdb.nix {};
   tests.docker = callTestOnTheseSystems ["x86_64-linux"] tests/docker.nix {};
+  tests.docker-tools = callTest tests/docker-tools.nix {};
```

Comment:
Should we perhaps do `tests.docker-tools = callTestOnTheseSystems ["x86_64-linux"] tests/docker-tools.nix {};` or `tests.docker-tools = callTestOnTheseSystems lib.platforms.linux tests/docker-tools.nix {};`

Reply:
To save on testing resources?